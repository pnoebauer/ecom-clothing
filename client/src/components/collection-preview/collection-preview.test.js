import React from 'react';
import {shallow} from 'enzyme';
import CollectionPreview from './collection-preview.component';

const mockProps = {
	routeName: 'hats',
	id: '5XnhLO3MJTywoST6lftW',
	title: 'Hats',
	items: [
		{
			id: 1,
			imageUrl: 'https://i.ibb.co/ZYW3VTp/brown-brim.png',
			price: 25,
			name: 'Brown Brim',
		},
		{
			imageUrl: 'https://i.ibb.co/ypkgK0X/blue-beanie.png',
			id: 2,
			name: 'Blue Beanie',
			price: 18,
		},
		{
			price: 30,
			imageUrl: 'https://i.ibb.co/0jqHpnp/sneakers.png',
			id: 3,
			name: 'sneakers',
		},
		{
			price: 10,
			imageUrl: 'https://i.ibb.co/GCCdy8t/womens.png',
			id: 4,
			name: 'womens',
		},
		{
			price: 5,
			imageUrl: 'https://i.ibb.co/R70vBrQ/men.png',
			id: 5,
			name: 'mens',
		},
	],
};

test('should render CollectionPreview component', () => {
	expect(shallow(<CollectionPreview {...mockProps} key={1} />)).toMatchSnapshot();
});
