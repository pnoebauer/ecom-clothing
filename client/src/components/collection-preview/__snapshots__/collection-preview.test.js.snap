// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CollectionPreview component 1`] = `
<collection-previewstyles__CollectionPreviewContainer>
  <collection-previewstyles__TitleContainer>
    Hats
  </collection-previewstyles__TitleContainer>
  <collection-previewstyles__PreviewContainer>
    <Connect(CollectionItem)
      item={
        Object {
          "id": 1,
          "imageUrl": "https://i.ibb.co/ZYW3VTp/brown-brim.png",
          "name": "Brown Brim",
          "price": 25,
        }
      }
      key="1"
    />
    <Connect(CollectionItem)
      item={
        Object {
          "id": 2,
          "imageUrl": "https://i.ibb.co/ypkgK0X/blue-beanie.png",
          "name": "Blue Beanie",
          "price": 18,
        }
      }
      key="2"
    />
    <Connect(CollectionItem)
      item={
        Object {
          "id": 3,
          "imageUrl": "https://i.ibb.co/0jqHpnp/sneakers.png",
          "name": "sneakers",
          "price": 30,
        }
      }
      key="3"
    />
    <Connect(CollectionItem)
      item={
        Object {
          "id": 4,
          "imageUrl": "https://i.ibb.co/GCCdy8t/womens.png",
          "name": "womens",
          "price": 10,
        }
      }
      key="4"
    />
  </collection-previewstyles__PreviewContainer>
</collection-previewstyles__CollectionPreviewContainer>
`;
